content/authors/Yiwei/index.md

```diff
@@ -0,0 +1,49 @@
+---
+# Display name
+title: Yiwei Huang
+
+# Major
+major: Statistics
+
+# Username (this should match the folder name)
+authors:
+- yiwei14
+
+# Is this the primary user of the site?
+superuser: false
+
+# Supervisor
+supervisor: Dr. Linglong Kong
+
+Organizations/Affiliations:
+organizations:
+- name: University of Alberta
+  url: "https://www.ualberta.ca/index.html"
+
+# Short bio (displayed in user profile at end of posts)
+bio: 
+
+interests:
+- power analysis
+- financial statistics
+
+
+education:
+  courses:
+  - course: statistics
+    institution: University of Alberta
+  - course: statistics and Economics
+    institution: University of Saskatchewan
+
+social:
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+email: "ruths@pims.math.ca"
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- project: BCFSA Project member
```

Comment:
I think this line is wrong, it should look like
```
user_groups:
- committee: members
  role: BCFSA Project member
```
The `committee: members` is important so that your profile shows up in the right section.